Microsoft.DotNet.Cli.sln

```diff
@@ -222,6 +222,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "templates", "templates", "{
 		build\templates\templates.csproj = build\templates\templates.csproj
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "dotnet-back-compat.Tests", "test\dotnet-back-compat.Tests\dotnet-back-compat.Tests.csproj", "{A4C198B4-D46E-4CA8-87DF-B2B206DCCAE6}"
```

Comment:
You need to add this test to this sln file as well: https://github.com/dotnet/cli/blob/master/test/Microsoft.DotNet.Cli.Tests.sln#L1.

The restore of the test projects happens based on the Tests.sln file.